docs/release-notes.md

```diff
@@ -16,6 +16,8 @@ nav_order: 9
 ### Changes
 
 - Warn if template for enabled systemd instance unit has no `Install` section
+- Warn when config indicates creating filesystem over whole-disk instead of a partition
```

Comment:
Not quite; this is legal if the whole disk device isn't being partitioned.  Maybe:

```suggestion
- Warn if filesystem overwrites partitioned disk
```